src/widgets/graph_widget.h

```diff
@@ -40,15 +40,15 @@ enum PerformanceHistoryGraphWidgets {
 
 /** Widget of the #PaymentRatesGraphWindow class. */
 enum CargoPaymentRatesWidgets {
-	WID_CPR_BACKGROUND,      ///< Background of the window.
-	WID_CPR_HEADER,          ///< Header.
-	WID_CPR_GRAPH,           ///< Graph itself.
-	WID_CPR_RESIZE,          ///< Resize button.
-	WID_CPR_FOOTER,          ///< Footer.
-	WID_CPR_ENABLE_CARGOES,  ///< Enable cargoes button.
-	WID_CPR_DISABLE_CARGOES, ///< Disable cargoes button.
-	WID_CPR_MATRIX,          ///< Cargo list.
-	WID_CPR_MATRIX_SCROLLBAR,///< Cargo list scrollbar.
+	WID_CPR_BACKGROUND,       ///< Background of the window.
+	WID_CPR_HEADER,           ///< Header.
+	WID_CPR_GRAPH,            ///< Graph itself.
+	WID_CPR_RESIZE,           ///< Resize button.
+	WID_CPR_FOOTER,           ///< Footer.
+	WID_CPR_ENABLE_CARGOES,   ///< Enable cargoes button.
+	WID_CPR_DISABLE_CARGOES,  ///< Disable cargoes button.
+	WID_CPR_MATRIX,           ///< Cargo list.
+	WID_CPR_MATRIX_SCROLLBAR, ///< Cargo list scrollbar.
```

Comment:
Does OpenTTD have any strict rules about formatting spaces? Seems like we have same spacing in other places so it's a little bit confusing

Reply:
In general yes, but you are fixing stuff that you otherwise don't touch -- that's why it's not necessary here.

It could be a separate commit that does just this if you like, though.